code/modules/xenoarcheaology/tools/tools.dm

```diff
@@ -310,3 +310,41 @@
 		usr << browse(null, "window=locater")
```

Comment:
to_chat

Reply:
I left this alone as I'm really not sure what the browse(null, "window=locator") does and didn't want to break it. I'll change it and see if it affects anything.

Reply:
Oh that's right, it's browse (I totally just saw the `usr<<`)